lib/scala/docs-generator/src/main/scala/org/enso/docsgenerator/DocsGenerator.scala

```diff
@@ -0,0 +1,63 @@
+package org.enso.docsgenerator
+
+import java.io.File
+import org.enso.syntax.text.{DocParser, Parser}
+import org.enso.syntax.text.docparser._
+
+/** The Docs Generator class. Defines useful wrappers for Doc Parser.
+  */
+object DocsGenerator {
+
+  /** Generates HTML of docs from Enso program.
+    */
+  def generate(program: String): String = {
```

Comment:
Call this `run` given the object is `DocsGenerator`. `DocsGenerator.run` is much nicer than `DocsGenerator.generate`.